### PR TITLE
add temurin 17 announcement

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

File: content/blog/eclipse-temurin-17-available/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+* [JEP 412:	Foreign Function & Memory API (Incubator)](https://openjdk.java.net/jeps/412).
+* [JEP 414:	Vector API (Second Incubator)](https://openjdk.java.net/jeps/414).
+* [JEP 415:	Context-Specific Deserialization Filters](https://openjdk.java.net/jeps/415).
+
+For a complete list of the enhancements (including ones that only impact developers of OpenJDK), [see the JDK 17 overview over at OpenJDK](https://openjdk.java.net/projects/jdk/17/).
+
+### Eclipse Temurin for Alpine Linux With musl libc 
+
+[Alpine Linux](https://alpinelinux.org/) is a popular Linux distribution for container workloads because of its small footprint. Contrary to most other Linux distributions, Alpine Linux is not based on the [C library created by the GNU project](https://www.gnu.org/software/libc/) (usually referred to as "glibc") but uses [musl libc](https://musl.libc.org) instead. So far, OpenJDK has not supported musl libc but only glibc. With Eclipse Temurin 17, this is no longer necessary. We now have separate variants of Eclipse Temurin 17 that are purpose-built for musl libc that can be downloaded as a tarball and are also available as ready-made container images.
```
Comment (tellison) [inline content/blog/eclipse-temurin-17-available/index.md:40]: Consider adding a link to the referenced musl builds/containers?

**Review by tellison — APPROVED** [review]

File: content/blog/eclipse-temurin-17-available/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Adoptium is happy to announce the immediate availability of Eclipse Temurin 17. As always, all binaries are available free of charge without usage restrictions on a wide range of platforms.
+
+* [List of changes in OpenJDK 17](https://bugs.openjdk.java.net/browse/JDK-8263045?jql=project%20%3D%20JDK%20AND%20fixVersion%20%3D%20%2217%22%20ORDER%20BY%20created%20DESC)
+
+## New and Noteworthy
+
+### Overview of Java 17 Features
+
+* [JEP 306: Restore Always-Strict Floating-Point Semantics](https://openjdk.java.net/jeps/306).
+* [JEP 356:	Enhanced Pseudo-Random Number Generators](https://openjdk.java.net/jeps/356).
```
Comment (karianna) [inline content/blog/eclipse-temurin-17-available/index.md:22]: whitespace weirdness?

**Review by karianna — CHANGES_REQUESTED** [review]

**Review by karianna — APPROVED** [review]

File: content/blog/eclipse-temurin-17-available/index.md (hunk 2 above)
Comment (gdams) [reply content/blog/eclipse-temurin-17-available/index.md]: fixed

File: content/blog/eclipse-temurin-17-available/index.md (hunk 1 above)
Comment (gdams) [reply content/blog/eclipse-temurin-17-available/index.md]: done

**Pull request merged**